assets/i18n/en.json

```diff
@@ -0,0 +1,177 @@
+      "FILTER": "Filter"
+    },
+    "DOCKER": {
+      "CONTAINER": "container",
+      "VOLUME": "volume",
+      "HOST": "host"
+    },
+    "UI": {
+      "LOADING": "Loading...",
+      "ALL": "All"
+    },
+    "PLACEHOLDERS": {
+      "USERNAME": "Username",
+      "CONTAINER_NAME": "e.g. web (optional)",
+      "CONTAINER_HOST_PORT": "e.g. 80 or 1.2.3.4:80 (optional)"
+    }
+  },
+  "WIDGETS": {
+    "HEADER_CONTENT": {
+      "USER_PREFERENCES": "my account",
+      "LOGOUT": "log out"
+    },
+    "PAGINATION": {
+      "TEXT": "Items per page:"
+    }
+  },
+  "AUTH": {
+    "LOGIN": "Login"
+  },
+  "SIDEBAR": {
+    "ACTIVE_ENDPOINT": {
+      "TITLE": "Active endpoint"
+    },
+    "ENDPOINT_ACTIONS": {
+      "TITLE": "Endpoint actions",
+      "DASHBOARD": "Dashboard",
+      "APP_TEMPLATES": "App Templates",
+      "SERVICES": "Services",
+      "CONTAINERS": "Containers",
+      "IMAGES": "Images",
+      "NETWORKS": "Networks",
+      "VOLUMES": "Volumes",
+      "SECRETS": "Secrets",
+      "EVENTS": "Events",
+      "SWARM": "Swarm",
+      "DOCKER": "Docker"
+    },
+    "PORTAINER_SETTINGS": {
+      "TITLE": "Portainer settings",
+      "USER_MANAGEMENT": {
+        "TITLE": "User management",
+        "TEAMS": "Teams"
+      },
+      "ENDPOINTS": "Endpoints",
+      "REGISTRIES": "Registries",
+      "SETTINGS": {
+        "TITLE": "Settings",
+        "AUTHENTICATION": "Authentication"
+      }
+    }
+  },
+  "TEMPLATES": {
+    "HEADER": {
+      "TITLE": "Application templates list",
+      "CONTENT": "Templates"
+    },
+    "SELECTED_TEMPLATE": {
+      "TITLE_INFORMATION": "Information",
+      "TITLE_CONFIGURATION": "Configuration",
+      "FORM": {
+        "NAME": "Name",
+        "NETWORK": "Network",
+        "SELECT_CONTAINER": "Select a container",
+        "SHOW_ADVANCED_OPTIONS": "Show advanced options",
+        "HIDE_ADVANCED_OPTIONS": "Hide advanced options",
+        "ADVANCED_OPTIONS": {
+          "PORT_MAPPING": {
+            "LABEL": "Port mapping",
+            "MAP_PORT": "map additional port",
+            "NOTE": "Portainer will automatically assign a port if you leave the host port empty.",
+            "HOST": "host"
+          },
+          "VOLUME_MAPPING": {
+            "LABEL": "Volume mapping",
+            "MAP_VOLUME": "map additional volume",
+            "NOTE": "Portainer will automatically create and map a local volume when using the <b>auto</b> option.",
+            "TYPE": {
+              "AUTO": "Auto",
+              "VOLUME": "Volume",
+              "BIND": "Bind"
+            },
+            "SELECT_VOLUME": "Select a volume",
+            "WRITABLE": "Writable",
+            "READONLY": "Read-only"
+          }
+        },
+        "ACTIONS": {
+          "SWARM_NOTE_NETWORK": "When using Swarm, we recommend deploying containers in a shared network. Looks like you don't have any shared network, head over the <a ui-sref='networks'>networks view</a> to create one.",
```

Comment:
Actually, these networks are referenced as "Swarm" networks in the API response. I'll change it to 

"When using Swarm, we recommend deploying containers in a Swarm network. Looks like you don't have any Swarm-scope network, head over the networks view to create one."